src/CookiePhysicsComponent.h

```diff
@@ -3,6 +3,13 @@
 
 #include "PhysicsComponent.h"
 
+typedef struct {
+    int hits;
+    double launchTime;
+    glm::vec3 launchPosition;
+    glm::vec3 hitPosition;
```

Comment:
Do we want to make this a list of positions hit? Consider a cookie bouncing between two houses, we increment `hits` but which `hitPosition` is marked?